frontend/src/components/HeaderNav.js

```diff
@@ -1,8 +1,14 @@
 import { Link } from "react-router-dom";
 import { useNavbar } from "../hooks/useNavbar";
+import Logout from "./Logout";
 
-export default function HeaderNav() {
+export default function HeaderNav({ isAuth }) {
     const { currentRoute, navigateTo } = useNavbar();
+
+    const handleLogout = () => {
+        localStorage.clear();
+        navigateTo("admin/login");
```

Comment:
Dont just clear the token.
You have logout api, use that.
Then dont use .clear() it will cause problems in the future if you have another data in localstorage. Just clear specific data